src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt

```diff
@@ -255,8 +262,14 @@ if [[ "$SINGLEJAR" == 1 ]]; then
 else
   # Create the shortest classpath we can, by making it relative if possible.
   RUNPATH="${JAVA_RUNFILES}/%workspace_prefix%"
-  RUNPATH="${RUNPATH#$PWD/}"
+  # Keep absolute paths on windows, so we can correctly support both ':' and ';' as separators
+  if ! is_windows; then
+    RUNPATH="${RUNPATH#$PWD/}";
+  fi
   CLASSPATH=%classpath%
+  if is_windows; then
+    CLASSPATH="$(sed -E 's/:([A-Z]):/;\1:/g' <<< "$CLASSPATH")"
```

Comment:
Could you add a comment explaining what this does?

Reply:
Done